docker-app/qfieldcloud/core/models.py

```diff
@@ -1218,6 +1226,49 @@ class Meta:
         default=False,
     )
 
+    def get_localized_datasets_project_id(self) -> str | None:
+        """
+        Returns the ID of the 'localized_datasets' project for the same owner.
+        """
+        project = Project.objects.filter(
+            name="localized_datasets", owner=self.owner
+        ).first()
+        return project.id if project else None
+
+    def get_missing_localized_layers(self) -> List[Dict]:
+        """
+        Identifies and returns a list of localized layer metadata dictionaries
+        that are not present in the associated localized_datasets project storage.
+
+        Returns:
+            List[Dict]: A list of layer dictionaries missing from the
```

Comment:
```suggestion
            A list of layer dictionaries missing from the
```

No need anymore, see https://github.com/opengisch/QFieldCloud/pull/1213

Reply:
Updated!